cinnabar/plotting.py

```diff
@@ -97,6 +98,8 @@ def _master_plot(
     statistic_type : str, default 'mle'
         the type of statistic to use, either 'mle' (i.e. sample statistic)
         or 'mean' (i.e. bootstrapped mean statistic)
+    marker_size : float, default 10
```

Comment:
I'm tempted to say that this point you really should be just passing in a dictionary of kwargs for matplotlib's scatter rather than incrementally adding new kwargs.

Reply:
Good point. I just pushed changes to address this suggestion. Let me know what you think